espnet2/bin/spk_embed_extract.py

```diff
@@ -128,6 +128,7 @@ def extract_embed(args):
             output_dir=args.output_dir,
             custom_bs=bs,
             average=args.average_embd,
+            embed_avg=args.embedding_average,
```

Comment:
ditto, how is `embed_avg` different from `average`?

Reply:
Yeah, this is something to clean up
Original functionality was different from what I wanted, and I didn't want to break existing recipes